### PR TITLE
Set up eightball command

### PR DESCRIPTION
**Pull request by naomi-lgbt**
To address #7:
Created array with the 20 responses from the original Magic 8 Ball.

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
image
Tested locally.

**Comment by naomi-lgbt** [comment]
image
Works on another channel as well.

**Comment by Twaha-Rahman** [comment]
LGTM!

File: src/commands/eightball.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ module.exports = {
    */
   command: function eightBall(message) {
     console.log('eightBall', message);
+    let options = ["As I see it, yes." , "Ask again later." , "Better not tell you now.", "Cannot predict now.", "Concentrate and ask again.", "Don't count on it.", "It is certain.", "It is decidedly so.", "Most likely.", "My reply is no.", "My sources say no.", "Outlook not so good.", "Outlook good.", "Reply hazy, try again.",
+"Signs point to yes.", "Very doubtful.", "Without a doubt.", "Yes.", "Yes - definitely.", "You may rely on it."]
+    let rand = Math.floor(Math.random() * 20)
```
Comment (bradtaniguchi) [inline src/commands/eightball.js]: Change `let` to `const` for both variable declarations. Generally don't use `let` if the variables aren't going to change.

File: src/commands/eightball.js [hunk 2]
```diff
@@ -8,7 +8,11 @@ module.exports = {
...
     // TODO:
```
Comment (bradtaniguchi) [inline src/commands/eightball.js]: Remove TODO

File: src/commands/eightball.js (hunk 1 above)
Comment (bradtaniguchi) [inline src/commands/eightball.js]: Change the `20` to `options.length`, this way if we add more options to the list we wont have to manually update the count.

**Review by bradtaniguchi — CHANGES_REQUESTED** [review]
Update a few things to help make the code easier to maintain.

File: src/commands/eightball.js (hunk 2 above)
Comment (naomi-lgbt) [reply src/commands/eightball.js]: Oops. :)

File: src/commands/eightball.js (hunk 1 above)
Comment (naomi-lgbt) [reply src/commands/eightball.js]: That's a good idea - I didn't anticipate that.

Comment (naomi-lgbt) [reply src/commands/eightball.js]: Makes sense to me.

**Comment by naomi-lgbt** [comment]
Proposed changes have been implemented and new commit generated.

**Review by bradtaniguchi — APPROVED** [review]
LGTM :+1:

**Pull request merged**